config/settings.yml

```diff
@@ -1453,8 +1453,10 @@ check_in:
     service_name: MAP-API
   travel_reimbursement_api:
     auth_url: https://login.microsoftonline.us
+    auth_url_v2: https://internal-dsva-vagov-staging-fwdproxy-1821450725.us-gov-west-1.elb.amazonaws.com:4483/
```

Comment:
looks good to me, if it's ok to include these URLs in settings.yml

Reply:
You don't need these URLs here since you defined them in the manifest repo [in this PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2811/files). You can do `auth_url_v2: ~`. Same with the one below.

Reply:
We fall back to these settings in vets-api if there's nothing defined in the [settings.local.yml section](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/dev/values.yaml#L1732-L2749) of the values file in manifest repo. But you do have those defined there, which is good.

Reply:
Groovy, just being defensive. Thanks for the tips 🙂